scapy/utils.py

```diff
@@ -1821,6 +1826,10 @@ def _read_block_idb(self, block, _):
         options_raw = self._read_options(block[8:])
         options = self.default_options.copy()  # type: Dict[str, Any]
         for c, v in options_raw.items():
+            if isinstance(v, list):
+                # If the option is a list, we take the last value
+                # (as per pcapng spec)
```

Comment:
Please quote the spec

Reply:
I believe Wireshark's behavior of displaying comments in order is due to its support for multiple comments, something Scapy lacks. We're adopting this behavior. For backward compatibility, however, when pkt.comment is used, we will still only reference the first item. I might have misunderstood the question, but does this sound right?